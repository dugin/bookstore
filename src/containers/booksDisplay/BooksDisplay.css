.BooksDisplay {
  min-height: calc(100vh - 140px);
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center; }
  .BooksDisplay__loading {
    color: #c33fa3; }
