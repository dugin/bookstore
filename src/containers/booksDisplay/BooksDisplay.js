import React, {Component} from 'react';


class BooksDisplay extends Component {

    render() {
        return (
            <div>

            </div>
        );
    }
}

export default BooksDisplay;