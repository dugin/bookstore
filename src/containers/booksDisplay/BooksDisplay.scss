@import '../../variables';

.BooksDisplay {
  min-height: calc(100vh - 140px);
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;

  &__loading{
    color: $color--accent;

  }
}