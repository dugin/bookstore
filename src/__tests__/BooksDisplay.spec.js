import React  from 'react';
import {shallow} from 'enzyme'
import BooksDisplay from '../containers/booksDisplay/BooksDisplay';
import toJson from 'enzyme-to-json';

describe('Books Display', () => {

    const booksDisplay = shallow(<BooksDisplay/>);

    it('renders properly', () => {
        expect(toJson(booksDisplay)).toMatchSnapshot();
    })
})