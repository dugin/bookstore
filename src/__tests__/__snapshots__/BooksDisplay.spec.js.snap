// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Books Display renders properly 1`] = `<div />`;
